docs/EN_US/ContainerizedHPCC/ContainerizedMods/LocalDeployment.xml

```diff
@@ -163,31 +185,287 @@ sasha-wu-archiver     ClusterIP    10.111.34.240   &lt;none&gt;       8877/TCP
 sql2ecl               LoadBalancer 10.107.177.180  localhost    8510:30054/TCP    2m6s
 dfs                   LoadBalancer 10.100.52.9     localhost    8520:30184/TCP    2m6s</programlisting></para>
 
-    <para>Notice the <emphasis role="strong">eclwatch</emphasis> service is
-    running on <emphasis role="strong">localhost:8010</emphasis>. Use that
-    address in your browser to access ECL Watch.</para>
+    <para>Locate the ECL Watch service and identify the EXTERNAL-IP and
+    PORT(S) for eclwatch. In this case, it is localhost:8010. </para>
 
-    <para>Inside ECL Watch, press the ECL button and go to the Playground
-    tab.</para>
+    <para>Open a browser and access ECLWatch, press the ECL button, and select
+    the Playground tab.</para>
 
     <para>From here you can use the example ECL or enter other test queries
     and pick from the available clusters available to submit your
     workunits.</para>
   </sect1>
 
   <sect1 id="terminatedefault">
-    <title>Terminate (Decommission) the system</title>
+    <title>Terminate (Decommission) the System</title>
 
     <para>To check which Helm charts are currently installed, run this
     command:</para>
 
     <para><programlisting>helm list</programlisting></para>
 
+    <para>This displays the installed charts and their names. In this example,
+    mycluster. </para>
+
     <para>To stop the HPCC Systems pods, use helm to uninstall:</para>
 
     <para><programlisting>helm uninstall mycluster</programlisting></para>
 
     <para>This stops the cluster, deletes the pods, and with the default
     settings and persistent volumes, it also deletes the storage used.</para>
   </sect1>
+
+  <sect1 id="PVCsLocal">
+    <title>Persistent Storage for a Local Deployment</title>
+
+    <para>When running on a single-node test system such as Docker Desktop,
+    the default storage class normally means that all persistent volume claims
+    (PVCs) map to temporary local directories on the host machine. These are
+    typically removed when the cluster is stopped. This is fine for simple
+    testing but for any real application, you want persistent storage.</para>
+
+    <para>To persist data with a Docker Desktop deployment, the first step is
+    to make sure the relevant directories exist:</para>
+
+    <orderedlist>
+      <listitem>
+        <para>Create data directories using a terminal interface:</para>
+
+        <para>For Windows, use this command:</para>
+
+        <para><programlisting>mkdir c:\hpccdata
+mkdir c:\hpccdata\dalistorage
+mkdir c:\hpccdata\hpcc-data
+mkdir c:\hpccdata\debug
+mkdir c:\hpccdata\queries
+mkdir c:\hpccdata\sasha
+mkdir c:\hpccdata\dropzone</programlisting></para>
+
+        <para>For macOS, use this command:</para>
+
+        <para><programlisting>mkdir -p /Users/myUser/hpccdata/{dalistorage,hpcc-data,debug,queries,sasha,dropzone}</programlisting></para>
```

Comment:
I'm curious, doesn't ~/hpccdata/... also work on OSX?

Reply:
It did not work when I tried it